src/test/lib/KArray.h

```diff
@@ -44,7 +44,7 @@ namespace wistd     // ("Windows Implementation" std)
 //
 
 template <class _Tp>
-inline constexpr
```

Comment:
These changes are unnecessary for templates because they're C++, right?

Reply:
Hmm, good question. I didn't think too hard about this, and I'm not a C++ expert. I can say the code compiles and tests seem to pass, but I can't say this change is necessary. I'm happy to revert here, since there was not affected inline.c file.